test/e2e/windows/host_process.go

```diff
@@ -479,11 +480,103 @@ var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] [Excluded:WindowsDoc
 		framework.ExpectEqual(p.Status.Phase, v1.PodSucceeded)
 	})
 
+	ginkgo.It("metrics should report count of started and failed to start HostProcess containers", func() {
+		ginkgo.By("Selecting a Windows node")
+		targetNode, err := findWindowsNode(f)
+		framework.ExpectNoError(err, "Error finding Windows node")
+		framework.Logf("Using node: %v", targetNode.Name)
+
+		ginkgo.By("Getting initial kubelet metrics values")
+		beforeMetrics, err := getCurrentHostProcessMetrics(f, targetNode.Name)
+		framework.ExpectNoError(err, "Error getting initial kubelet metrics for node")
+		framework.Logf("Initial HostProcess container metrics -- StartedContainers: %v, StartedContainersErrors: %v, StartedInitContainers: %v, StartedInitContainersErrors: %v",
+			beforeMetrics.StartedContainersCount, beforeMetrics.StartedContainersErrorCount, beforeMetrics.StartedInitContainersCount, beforeMetrics.StartedInitContainersErrorCount)
+
+		ginkgo.By("Scheduling a pod with a HostProcess init container that will fail")
```

Comment:
it looks like we don't schedule a successful pod for this test?  Is that something we should be doing?

Reply:
I don't think we need to for this test case.
`StartedHostProcessContainersTotal` increments when the containers are started (regardless of outcome) and `StartedHostProcessContainersErrorsTotal` increments when containers fail to start.
Starting a pod with an init container that fails and another pod with a normal container that fails should cover all the test cases.
As mentioned in the code I was trying to avoid looking for discreet numbers in these metrics to avoid putting it in the `[Serial]` bucket of test cases.
Perhaps we can add a another serial test case that does check discreet values?
Do you mind if that comes in as another PR after this one?

Reply:
Also i forgot to mention that the behaviors for the two metrics mirror the existing kubelet behaviors for non-hostprocess pods. I think we should keep those behaviors consistent.